lib/MqttClient.js

```diff
@@ -416,4 +416,13 @@ MqttClient.prototype.handleCustomCommand = function (message) {
     }
 };
 
+MqttClient.prototype.shutdown = function() {
+    console.debug("Shutting down the MQTT Client...")
+    
+    //force close the mqtt client
+    this.client.end(true);
```

Comment:
https://github.com/mqttjs/MQTT.js#end

This can accept a callback which we might want to wait for

Reply:
yep. added it in the second commit